src/pybind/mgr/dashboard/controllers/orchestrator.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+import cherrypy
+
+from . import ApiController, Endpoint, ReadPermission
+from . import RESTController, Task
+from ..security import Scope
+from ..services.orchestrator import OrchClient
+from ..tools import wraps
+
+
+def orchestrator_task(name, metadata, wait_for=2.0):
+    return Task("orchestrator/{}".format(name), metadata, wait_for)
+
+
+def raise_if_no_orchestrator(method):
+    @wraps(method)
+    def inner(self, *args, **kwargs):
+        orch = OrchClient.instance()
+        if not orch.available():
+            raise cherrypy.HTTPError(503)
+        return method(self, *args, **kwargs)
+    return inner
+
+
+@ApiController('/orchestrator')
+class Orchestrator(RESTController):
+
+    @Endpoint()
+    @ReadPermission
+    def status(self):
+        return OrchClient.instance().status()
+
+
+@ApiController('/orchestrator/inventory', Scope.HOSTS)
```

Comment:
hm, what do you think of merging the orchestrator inventory with `ceph device ls`? similar to hosts?

Reply:
@votdev raise this question before. There is an issue for tracking `device ls` feature: https://tracker.ceph.com/issues/39352 and it is still pending for discussion.